### DISCUSSION TITLE
[Feature Request]: 希望mp的开发者能关注下，看看能不能将这个议题内的功能，做为插件或者内置到主程序中

### DISCUSSION THREAD

**Discussion by 99Mr-Z** [jxxghp/MoviePilot#3348] [讨论]
### 当前程序版本

1.72

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

希望mp的开发者能关注下，看看能不能将这些功能，做为插件或者内置到主程序中。

1.用户可以通过插件或者主程序的功能进行一个批量的识别（仅将文件识别的结果展示出来）
2.允许用户直接在插件或者程序中直接进行编辑，对无法识别媒体的的文件名，可以直接输入需要转义的文件名。
3.允许将识别的所有的结果生成一个excel表格、word文件、txt文件等比较常见的文件格式，便于分享或添加到共享词库。
4.条件允许可以将这个功能模块单独制作为一个Windows上的运行程序，便于编辑

**Comment by 99Mr-Z** [comment]
有这个想法主要原因是担忧媒体库的丢失损坏，如果媒体库辛苦搭建完成后，出现意外导致媒体库文件全部丢失，想要重新搭建媒体库需要花费较多的重复工作及时间，如果可以以上提到的功能能够实现做到的情况下，后续可以通过仅备份种子文件以及定期做个维护测试，就能通过备份配置文件、种子文件、词库这样轻量的备份方式，快速恢复媒体库的搭建，而不必采用额外的硬盘来定期做冷备份。
其次一些动漫相关的资源由于命名并不太规范，主要是一些特典资源或者冷门资源，同时大部分搭建影音库可能更多以收录BD资源为主，通常会包含特典文件，若只是简单粗暴的屏蔽掉特典文件的路径，收录多多少少有点缺失，如果能够实现以上功能，能扩大识别的范围，也便于搭建更加完善的动漫媒体库，同时便于用户操作，将识别过的文件分享到共享词库，便于词库的壮大，进一步提高识别效率和准确性。

**Comment by SoTender** [comment]
这不是emby就有的功能

**Comment by 99Mr-Z** [comment]
> 这不是emby就有的功能

能否指点一二，整个媒体库大约有25T左右了，不太想再花钱买两块16T的盘只为了做冷备份，过于浪费了

**Comment by 99Mr-Z** [comment]
> 这不是emby就有的功能

目前有一个4T的盘能做冷备份，请教下如何实现只利用这一块4T的盘来快速恢复25T的媒体库？把种子、字幕文件做个压缩包，差不多也就5个G左右

**Comment by autunn** [comment]
> > 这不是emby就有的功能
> 
> 目前有一个4T的盘能做冷备份，请教下如何实现只利用这一块4T的盘来快速恢复25T的媒体库？把种子、字幕文件做个压缩包，差不多也就5个G左右

在emby里面直接备份媒体库（前提是你勾选了将刮削文件备份到文件夹）这个文件夹里面就有你所想要备份的文件

**Comment by 99Mr-Z** [comment]
> > > 这不是emby就有的功能
> > 
> > 
> > 目前有一个4T的盘能做冷备份，请教下如何实现只利用这一块4T的盘来快速恢复25T的媒体库？把种子、字幕文件做个压缩包，差不多也就5个G左右
> 
> 在emby里面直接备份媒体库（前提是你勾选了将刮削文件备份到文件夹）这个文件夹里面就有你所想要备份的文件

4T的盘能压缩备份大约26T的媒体库？
